assets/js/googlesitekit/datastore/user/feature-tours.test.js

```diff
@@ -286,4 +350,51 @@ describe( 'core/user feature-tours', () => {
 			} );
 		} );
 	} );
+
+	describe( 'getLastDismissedAt', () => {
+		it( 'returns initial state (undefined) if there is no lastDismissedAt timestamp', () => {
+			const lastDismissedAt = registry.select( STORE_NAME ).getLastDismissedAt();
+			expect( lastDismissedAt ).toEqual( undefined );
+		} );
+
+		it( 'returns the lastDismissedAt timestamp if there is one', () => {
+			const timestamp = Date.now();
+			registry.dispatch( STORE_NAME ).receiveLastDismissedAt( timestamp );
+			expect( registry.select( STORE_NAME ).getLastDismissedAt() ).toEqual( timestamp );
+		} );
+
+		it( 'sets lastDismissedAt in the store if there is a value in the cache', async () => {
```

Comment:
Let's clarify that this is a side-effect performed by its resolver rather than the selector itself.